.github/actions/docusaurus-swizzled-warning/index.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+const core = require('@actions/core')
+const github = require('@actions/github')
+const fetch = require('node-fetch')
+const {
+  getAllFilesForPullRequest,
+  getChangedFilesBetweenTags,
+  findKeyEndingWith,
+} = require('./helpers')
+
+async function run() {
+  try {
+    const token = core.getInput('github-token', { required: true })
+
+    const { pull_request: pr } = github.context.payload
+    if (!pr) {
+      throw new Error('Event payload missing `pull_request`')
+    }
+
+    const client = github.getOctokit(token)
+    const packageName = 'docusaurus-theme-openapi'
+    const packageParentName = 'docusaurus-preset-openapi'
+    const overideComponents = ['Curl', 'Response']
+    const messageTemplate = `<table><thead><tr><th>
```

Comment:
Minor points, but:

1. If we have a look at your screenshot, we can see the table looks wierd because not all of the table rows have the same number of columns. Making these headers <th colspan="2"> will fix that.

![Screenshot at 2023-08-12 19-19-02](https://github.com/badges/shields/assets/6025893/40323a20-3c62-492c-98ce-894f66b903e4)

2. We're using `<th>` for every cell in this table, but `<th>` is only for headers. The "normal" cells should be `<td>`s.

Reply:
Updating this, not sure what you mean at 1, but i used `colspan="2"` to fix the visual issue.

Reply:
After testing
![image](https://github.com/badges/shields/assets/15849761/c1bf3da0-63d5-4caf-a6f8-b231cd18eac2)